features/candidates/schools/search/results/sorting.feature

```diff
@@ -15,6 +14,17 @@ Feature: Schools search page sorting
         When I select 'Distance' in the 'Sorted by' select box
         Then the results should be sorted by distance, nearest to furthest
 
+    @javascript
+    Scenario: Sorting by distance when searching by current location
+        Given there there are schools with the following attributes:
+            | Name              | Fee | Location   |
+            | Manchester School | 30  | Manchester |
+            | Rochdale School   | 10  | Rochdale   |
+            | Burnley School    | 20  | Burnley    |
+        And I have provided a point in 'Bury' as my location
+        When I select 'Distance' in the 'Sorted by' select box
```

Comment:
I would take into consideration that 'Distance' is defaulted as a search choice and so filtering by 'Name' and then filtering back to 'Distance' would be better coverage for these scenarios.

Reply:
*there

Reply:
line 19 - there*